applications/fishing-map/src/features/map/map.slice.ts

```diff
@@ -91,6 +91,7 @@ type SublayerVessels = {
 
 export const fetch4WingInteractionThunk = createAsyncThunk<
   { vessels: SublayerVessels[] } | undefined,
+  // TODO the whole function could be greatly simplified if only one temporalGridFeature was accepted, which is effecttively always the case
```

Comment:
What I meant was that we only need an array of datasets, all other parameters have to be the same